app/controllers/foreman_salt/minions_controller.rb

```diff
@@ -6,14 +6,21 @@ class MinionsController < ::ForemanSalt::ApplicationController
     add_smart_proxy_filters :node, :features => 'Salt'
 
     def node
-      enc = {}
+      enc = {'foreman' => {}}
```

Comment:
Ah ok, I see.  Yea, it's using ext_pillar.  We could make that easier to see that's pillars, the underlying format actually comes from Puppet (the idea from the Saltstack devs was to be able to reuse some existing puppet ENC's).

I think that's easy enough to do.  

The YAML change is fine if you want to make the PR just do that, I bet a lot of people would be interested.

Thanks!